powdermonkey/src/configuration/products.json

```diff
@@ -37,5 +44,32 @@
                 "url": "https://www.tokensets.com/portfolio/ethmaxy"
             }
         ]
+    },
+    "bye": {
+        "name": "Basis Yield ETH Index (BYE)",
+        "description": "BYE is the best Basis Yield $ETH Index strategy in DeFi today, all within one tradable ERC20 token.",

```

Comment:
@duck-0x happy with this description? Taken from documentation page but it's almost identical to ETHMAXY

Reply:
maybe lets use this @Crown-dev 
"BYE is an ERC20 composable structured product that executes a high yield, cash-and-carry delta-neutral strategy."